app/AccountancyModule/PaymentModule/EducationModule/presenters/SelectForGroupPresenter.php

```diff
@@ -6,12 +6,12 @@
 
 use App\AccountancyModule\PaymentModule\BasePresenter;
 use Cake\Chronos\ChronosDate;
-use Model\Payment\ReadModel\Queries\EducationsWithoutGroupQuery;
+use Model\Event\ReadModel\Queries\EducationListQuery;
 
 class SelectForGroupPresenter extends BasePresenter
 {
     public function actionDefault(): void
     {
-        $this->template->setParameters(['educations' => $this->queryBus->handle(new EducationsWithoutGroupQuery(ChronosDate::today()->year))]);
+        $this->template->setParameters(['educations' => $this->queryBus->handle(new EducationListQuery(ChronosDate::today()->year))]);
```

Comment:
Není tady problém, že ti to stejně vždy vezme jen udalosti z aktuálního roku?

Reply:
Jo, vezme, viz #2622